matpower/matpower-converter/src/main/java/com/powsybl/matpower/converter/MatpowerImporter.java

```diff
@@ -124,7 +121,8 @@ private static void createBuses(MatpowerModel model, ContainersMapping container
     }
 
     private static void createGenerators(MatpowerModel model, MBus mBus, VoltageLevel voltageLevel) {
-        model.getGenerators().stream().filter(gen -> gen.getNumber() == mBus.getNumber()).forEach(mGen -> {
+        MGen mGen = model.getGeneratorByBusNum(mBus.getNumber());
```

Comment:
Just a question: is it possible to have several generators to the same bus?

Reply:
It's seems yes, even if not seen in our cases. Thanks I'm going to fix it.

Reply:
Fixed